constellations/gemini-admin/src/ask_for_help/afh.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import moment from "moment";
+import API from "../api.js";
+import { Link } from "react-router-dom";
+
+export class AskForHelpPage extends React.Component {
+    state = {
+        list: [],
+    };
+
+    componentDidMount() {
+        /*let data = {
+            id: 1,
+            title: "AP Calculus Help",
+            date: new Date(),
+            timeString: "2:00-4:00PM",
+            subject: "AP Calculus",
+            locationId: "wchs",
+            notes: "test note"
+        }
+        API.post("api/askforhelp/create", data).catch((err) => alert(err))*/
+
+        API.get("api/askforhelp/all").then((res) => {
+            const afh = res.data;
+            this.setState({
+                list: afh,
+            });
+            console.log(this.state.list);
+        });
+    }
+
+    render() {
+        const rows = this.state.list.map((row, index) => {
+            return <AFHRow row={row} key={index} />;
+        });
+
+        return (
+            <div id="view-afh">
+                <h1>All Ask For Help Sessions</h1>
+
+                <ul id="header">
+                    <li className="li-small">Date</li>
+                    <li className="li-med">Time</li>
+                    <li className="li-med">Title</li>
+                    <li className="li-med">Subject</li>
+                    <li className="li-small">LocationId</li>
+                    <li className="li-large">Notes</li>
```

Comment:
Do you think it would make more sense to have the Title first? (I'm not sure which order would be better)

Reply:
I'm not completely sure on what the order should be, I just used the order that they were listed in the Asana task

Reply:
On second thought, let's reorder:

Date
Time
Subject
Title
LocationId
Notes

To me, I'm thinking of viewing things like a spreadsheet, and I usually like date/time stuff on the leftmost. Then from generic stuff to specific stuff (i.e. "Math" as "Subject", "AP Calculus Practice Exam" for Title, then location, etc.)

But of course, happy to get any feedback / suggestions if what I described feels weird!